### ISSUE TITLE
I have a question for the creator of this library

### ISSUE THREAD

**Issue by KULTI1995** [nativewind/nativewind#928]
How can we trust you as a creator if, for the third time, the same thing happens as with versions 2 and 3. You start another version again, warm up people, and then stop the implementation. Next will be version 5 and the unfinished version 4?

**Comment by Bahnasawy** [comment]
It may not be my place to answer this, but the creator abandoned v3 for v4 for several (valid) reason. But if you're looking for a guarantee, he is now employed by Expo, so there is now no reason or doubt for this to happen again.

**Comment by tyrauber** [comment]
Not my place either, but v4 is not unfinished.  V3 was abandoned for v4, because of changes to React Native to enable implementing css at a lower level. While there are some edge cases and bugs, v4 is very much usable, and there are apps using it in production.

**Comment by vendramini** [comment]
I personally don't recommend using this library, although I love the idea and I've already used it.

I started 2 projects at the same time with it, and I got in some problems with one of them. I did everything I could to try to solve it, but I couldn't - I'm not going deeper into it, it's not relevante. The fact is that: I lost days trying to use it in a stable way without success - it worked out for a period, then when you update something it just stops working. Specially if you are on a complex mono repo, like were/are my case.

Before I got into the same problem with the other project, I decided to rewrite both of them using https://www.unistyl.es/
Of course you will code a bit more and lose the practicality of className='xxx', but at least you have something stable to ship to production which has been maintained.
